projects/arcus.cmake

```diff
@@ -1,9 +1,3 @@
-if(BUILD_OS_LINUX)
-    set(arcus_cmake_command PATH=${CMAKE_INSTALL_PREFIX}/bin/:$ENV{PATH} LD_LIBRARY_PATH=${CMAKE_INSTALL_PREFIX}/lib/ PYTHONPATH=${CMAKE_INSTALL_PREFIX}/lib/python3/dist-packages/:${CMAKE_INSTALL_PREFIX}/lib/python3.5:${CMAKE_INSTALL_PREFIX}/lib/python3.5/site-packages/ ${CMAKE_COMMAND})
```

Comment:
Wait, so how does it install the Python bindings to the correct location now?

Reply:
Yes, it does. This is done via `PATH` and `PKG_CONFIG_PATH` that's set before `cmake` is run.  Variables such as `CMAKE_PREFIX_PATH` also provides necessary context in this case.